src/tools/illink/test/Mono.Linker.Tests.Cases/Attributes/IVTUnused.cs

```diff
@@ -1,15 +1,16 @@
-using System.Runtime.CompilerServices;
 using Mono.Linker.Tests.Cases.Expectations.Assertions;
 using Mono.Linker.Tests.Cases.Expectations.Metadata;
 
 #if IVT
-[assembly: InternalsVisibleTo ("missing")]
-[assembly: InternalsVisibleTo ("test-with-key, PublicKey=00240000")]
-
+[assembly: System.Runtime.CompilerServices.InternalsVisibleTo ("missing")]
+[assembly: System.Runtime.CompilerServices.InternalsVisibleTo ("test-with-key, PublicKey=00240000")]
 #endif
 
+[assembly: KeptAttributeAttribute ("System.Runtime.CompilerServices.InternalsVisibleToAttribute")]
+
 namespace Mono.Linker.Tests.Cases.Attributes
 {
+	[SetupLinkerArgument("--used-attrs-only", "true")]
```

Comment:
@sbomer Not necessary to get the test passing, however needed to ensure that the changes to `ShouldMarkCustomAttribute` in the case of `--used-attrs-only` being passed works correctly: https://github.com/dotnet/runtime/pull/98910/files#diff-173496ee0ea5037ffe1a0f2cae5ad637aee0016745c24cfb24ca081b8dd58608L1156-L1173

Otherwise the IVT may still be stripped when `--used-attrs-only` is passed.

So I've split this into two tests so that it confirms the IVT is retained regardless of if `--used-attrs-only` is passed.